x11-themes/terminology-themes/terminology-themes-9999.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Color schemes for the Terminology terminal emulator"
+HOMEPAGE="https://charlesmilette.net/terminology-themes/"
+SRC_URI=""
+
+if [[ ${PV} == 9999 ]]; then
```

Comment:
Thanks for the PR. We avoid providing live ebuilds only for a given package and always strive to provide users with at least a set version.